person_with_age.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age = 0, parent_permission: true, name: 'Unknown')
+    super()
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
Whenever I try to create a person Im encountering an error. It would be nice if you fixed this to enable creation of a person. 

Here is the error:
![Screenshot 2023-11-25 at 6 10 25 PM](https://github.com/BHObserver/OOP-School-Library/assets/79506105/71e949d5-b3bb-47dc-862c-1b76addea6d9)

![Screenshot 2023-11-25 at 6 10 33 PM](https://github.com/BHObserver/OOP-School-Library/assets/79506105/28f7416c-4ad5-4e01-a3b0-c7c211abc2f7)